pkg/manilaapi/statefulset.go

```diff
@@ -70,6 +72,46 @@ func StatefulSet(
 			Port: intstr.IntOrString{Type: intstr.Int, IntVal: int32(manila.ManilaPublicPort)},
 		}
 		readinessProbe.HTTPGet = livenessProbe.HTTPGet
+
+		if instance.Spec.TLS.API.Enabled(service.EndpointPublic) {
+			livenessProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+			readinessProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+		}
+	}
+
+	// create Volume and VolumeMounts
+	volumes := GetVolumes(manila.GetOwningManilaName(instance), instance.Name, instance.Spec.ExtraMounts)
+	volumeMounts := GetVolumeMounts(instance.Spec.ExtraMounts)
+
+	// Add log volume and volume mount
+	logVolume := GetLogVolume()
+	logVolumeMount := GetLogVolumeMount()
```

Comment:
seems to be not used in the podspec? should be used for the `Name: instance.Name + "-log",` container in L160?